core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java

```diff
@@ -186,6 +186,7 @@ public static boolean isExpected(Throwable cause) {
      * Returns {@code true} if the specified exception will cancel the current request or response stream.
      */
     public static boolean isStreamCancelling(Throwable cause) {
+        // TODO(minwoox): return true if the cause is "io.grpc.StatusRuntimeException: CANCELLED"
```

Comment:
Maybe file a new issue scheduled for the next milestone?

Reply:
Created. https://github.com/line/armeria/issues/5254 😉

Reply:
Thanks!